### DISCUSSION TITLE
Demo application

### DISCUSSION THREAD

**Discussion by kevinpapst** [kevinpapst/TablerBundle#35] [Ideas]
There is a demo application of this bundle existing at https://github.com/kevinpapst/TablerBundle-Demo

Install it: eg. with:
```
git clone git@github.com:kevinpapst/TablerBundle-Demo.git
cd TablerBundle-Demo
composer install
symfony serve
```

If you want to use your own branch of the TablerBundle adjust composer.json. First, point to your branch:
```
    "require": {
        "kevinpapst/tabler-bundle": "dev-feature/button-embed",
    },
```
and then add your repo:
```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/cavasinf/TablerBundle"
        }
    ],
```
A `composer update` will now install the branch.

**Comment by cavasinf** [comment]
Wait, it's in Symfony 5.

Missing file `bin/console`
And folder `translations`

**Reply by kevinpapst** [reply]
`git pull` please

**Comment by cavasinf** [comment]
Got a 404 and no debugger/dev mode.

![image](https://user-images.githubusercontent.com/25293190/146577005-317bb35f-3e48-47a6-9cd6-f8922d86a5bc.png)

But default homepage symfony works fine
![image](https://user-images.githubusercontent.com/25293190/146577136-ce30c5ac-ec0d-4bf0-a4d1-9b532152bfd2.png)
(That's mean Symfony doesn't recognize `homepage` route name)

![image](https://user-images.githubusercontent.com/25293190/146577224-862f7c03-f64a-404e-b923-c5e2762fe1e6.png)

**Reply by kevinpapst** [reply]
`git pull` again and run `composer update kevinpapst/tabler-bundle` as well.

There was a locale listener for the default route, which did not trigger, seems something changed between SF4 and 5.

**Reply by cavasinf** [reply]
MANY things has changed between 4 & 5. Not from App files but Symfony conf.

I can give you the code/git.zip of my SF5 with Tabler if you want.

**Reply by kevinpapst** [reply]
It works for me and @tacman - so I guess the worst problems are fixed.
But yes please, everything that makes the demo and theme better is welcome as PR!

**Reply by cavasinf** [reply]
Still have 404 on my side 😢

**Reply by kevinpapst** [reply]
Weird, you could try again. I think I fixed all SF5 deprecations and upgraded composer packages.

**Comment by cavasinf** [comment]
I've used Symfony demo to upgrade my SF4.
https://github.com/symfony/demo/tree/v1.8.0
Helped me quite a lot.

Php 7.4 is deprecated in 2022. Wouldn't it be worth it to have a TablerBundle demo with PHP 8 ?
Symfony demo is also in PHP 8.
